contracts/reverseRegistrar/ReverseRegistrar.sol

```diff
@@ -13,11 +15,20 @@ bytes32 constant lookup = 0x3031323334353637383961626364656600000000000000000000
 
 bytes32 constant ADDR_REVERSE_NODE = 0x91d1777781884d03a6757a803996e38de2a42967fb37eeaca72729271025a9e2;
 
+bytes4 constant claimForAddrWithSignatureFuncId = bytes4(
```

Comment:
You should be able to do this by declaring an interface, then using `interface.function.selector`.